src/handlers/outbound/linking/request-consent/{ID}/pass-credential.ts

```diff
@@ -0,0 +1,82 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the "License")
+ and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed
+ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Kevin Leyow <kevin.leyow@modusbox.com>
+ --------------
+ ******/
+
+import { StateResponseToolkit } from '~/server/plugins/state'
+import { Request, ResponseObject } from '@hapi/hapi'
+import { PISPLinkingModel, loadFromKVS } from '~/models/outbound/pispLinking.model';
+import {
+  PISPLinkingModelConfig,
+} from '~/models/outbound/pispLinking.interface'
+import config from '~/shared/config';
+import inspect from '~/shared/inspect';
+import * as OutboundAPI from '~/interface/outbound/api_interfaces'
+
+
+/**
+ * Handles outbound POST /linking/request-consent/{ID}/pass-credential request
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+// todo: consider changing this to PUT /linking/request-consent/{ID}/pass-credential
+//       since the flow triggers a PUT /consents/{ID}
+async function post (_context: any, request: Request, h: StateResponseToolkit): Promise<ResponseObject> {
```

Comment:
we can drop using `any` and linter exception if we will use 'unknown' instead

Reply:
I'll get in a follow up. copy/paste sry.